client/paper/whisker.js

```diff
@@ -0,0 +1,171 @@
+import EventEmitter from 'events';
```

Comment:
I've used the node EventEmitter because the browser event system adds unnecessary complication like event bubbling and dealing with Event objects instead of plain JS objects.

Reply:
That makes sense, on the other hand this does add a non-standard API into Paper Programs. I think I would slightly prefer to use `EventTarget` anyway because it's an accepted browser standard. What do you think?

Reply:
Yeah that makes sense too. I'm mostly concerned about future compatibility, and bringing in concepts from Node feels a bit heavy in that respect. I don't have a super strong opinion though, so also happy to keep it this way.

And yeah, that would be useful! I don't really have goals for Paper Programs other than where you and other contributors want to take it. :)